docs/ref/forms/fields.txt

```diff
@@ -585,6 +585,11 @@ For each field, we describe the default widget used if you don't specify
     If provided, these arguments ensure that the string is at most or at least the
     given length.
 
+    .. versionadded:: 1.11
```

Comment:
I don't think it's important to mention the issue here since the option was never documented. A mention in the 1.11 release notes should be sufficient.